regression/tests/studio/test_course_pages.py

```diff
@@ -0,0 +1,54 @@
+"""
+Course pages test
+"""
+from uuid import uuid4
+from bok_choy.web_app_test import WebAppTest
+
+from regression.pages.studio.login_studio import StudioLogin
+from regression.tests.helpers import LoginHelper, get_course_info
+from regression.pages.studio.pages_page_studio import PagesPageExtended
+
+
+class CoursePagesTest(WebAppTest):
+    """
+    Course Pages test
+    """
+    def setUp(self):
+        super(CoursePagesTest, self).setUp()
+        self.course_info = get_course_info()
+        self.login_page = StudioLogin(self.browser)
+        LoginHelper.login(self.login_page)
+        self.pages_page = PagesPageExtended(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+    def test_pages(self):
+        """
+        Tests basic scenarios related to pages section of course.
+        Scenarios covered are adding, editing and
+        deleting pages.
+        """
```

Comment:
It would help future contributors if we follow BDD style docstrings for tests. Please see docstings of below tests as for examples:

https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/video/test_video_module.py#L494
https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/video/test_video_module.py#L980